FairRoot_doxygen_test.cmake

```diff
@@ -42,6 +33,32 @@ list(APPEND options
 )
 ctest_configure(OPTIONS "${options}")
 
-ctest_build(TARGET DoxygenDoc FLAGS "")
+ctest_build(TARGET DoxygenDoc FLAGS ""
+            NUMBER_ERRORS _ctest_build_num_errs
+            RETURN_VALUE _ctest_build_ret_val)
+
+if(_ctest_build_ret_val OR _ctest_build_num_errs)
+  message(STATUS "Skipping Publishing, because build failed"
+          " (return value: ${_ctest_build_ret_val},"
+          " number of errors: ${_ctest_build_num_errs})")
+else()
+  if (DEFINED ENV{CHANGE_ID} AND NOT ("$ENV{BUILD_TAG}" STREQUAL ""))
```

Comment:
Now that I see this, I am thinking whether we should better not hardwire jenkins variables here, but define an input parameter, e.g. `IS_PR` and set it from the jenkins script?

Reply:
Alternatively, lets put at least a comment that explains what those environment variables mean and where they come from?